### DISCUSSION TITLE
Falcon-RX switch alternative

### DISCUSSION THREAD

**Discussion by fllay** [srsran/srsRAN_Project#889] [Ideas & Suggestions]
Is it possible to replace the Falcon-RX with something like 

https://store.timebeat.app/products/open-timecard ?

it has a 10G network interface with ptp. Can I just put it in a server running srsRAN and connect split 7.2 RU to that 10G network interface? Or is it  better to  use  µFalcon-RX?

BTW  do I need Rb clock or just OCXO is ok? GNSS signal where I live is good.

**Comment by andrepuschmann** [comment]
No, the PHY on the Timecard is just a Intel i226 I think, it's a 1G Ethernet PHY only. The SFP cage I guess is just for convenience.
However, Timebeat also has slightly modified [Intel E810s](https://store.timebeat.app/products/clock-sync-network-adapter-with-1pps-e810-xxvda2) that allow external PPS input from e.g. the Timecard or another PPS source. Those NICs work well as OFH NICs.

**Reply by fllay** [reply]
Thank you for information. With this NIC

https://store.timebeat.app/products/clock-sync-network-adapter-with-1pps-e810-xxvda4

it should be ok right?

**Reply by andrepuschmann** [reply]
If you use a normal E810 or this modified version you need to tame the PHC on the card with an external PPS signal. The adapter just provides you a way to connect it over SMA.
The other alternative is to get the E810 with GPS receiver mounted on the card https://cdrdv2-public.intel.com/641626/Intel%20Ethernet%20Network%20Adapter%20E810-XXVDA4T%20Product%20Brief.pdf

**Reply by fllay** [reply]
Thank you very much. I got the idea.
